### DISCUSSION TITLE
Shift + Numpad

### DISCUSSION THREAD

**Discussion by Barnacle** [Jagailo/YetAnotherKeyDisplayer#14] [Ideas]
The app doesn't handle properly Numpad keys with Shift modifier.

Steps to reproduce:

1. Press and hold Shift key
2. Press and hold num key (Num5 for example)
3. Release Shift key
4. Release num key

The result will look like this:

1. L Shift
2. Clear
3. Numpad 5 + Clear
4. Clear (stucked)

**Comment by Jagailo** [comment]
Hello, thank you for creating an issue.
I'll check and try to fix it as soon as I can.

**Comment by Jagailo** [comment]
The combination of Shift and Num5 is defined by Windows as pressing a separate key.
I have no idea how to distinguish between Shift + Num5 from Clear.

I checked the [flags](https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-kbdllhookstruct#members), hoping there would be something like the 5th bit that gets activated when the Alt key is pressed, but no.

I'll leave this issue open, maybe someone will figure out what to do about it.
